Features/FindDataById/testPages/data/find_data_page.py

```diff
@@ -0,0 +1,173 @@
+
+import time
+from selenium.webdriver.common.by import By
+from Features.FindDataById.userInputs.user_inputs import UserData
+from common_utilities.selenium.base_page import BasePage
+
+
+""""Contains test page elements and functions related to the Lookup Table module"""
+
+class FindIdPage(BasePage):
```

Comment:
Keep the class name same as the file name. Like your file name is "find_data_page.py" so your class name should be "FindDataPage" or vice versa.